app/services/classification/base.rb

```diff
@@ -160,6 +160,12 @@ def get_element_and_element_name(lookup_value)
       element_name = ''
       lookup_value.match.split(LookupKey::KEY_DELM).each do |match_key|
         lv_element, lv_element_name = match_key.split(LookupKey::EQ_DELM)
+        case lv_element
```

Comment:
Yeah, I understand.  I don't mind if you want to address it (separately?), but I'd have more comments - since we'd need to translate everything, not just these two resource types.